pkgs/applications/networking/aether/default.nix

```diff
@@ -0,0 +1,89 @@
+{ autoPatchelfHook, makeDesktopItem, lib, stdenv, wrapGAppsHook
+, alsa-lib, at-spi2-atk, at-spi2-core, atk, cairo, cups, dbus, expat, fontconfig
+, freetype, gdk-pixbuf, glib, gtk3, libcxx, libdrm, libnotify, libpulseaudio, libuuid
+, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
+, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, libxshmfence
+, mesa, nspr, nss, pango, systemd, libappindicator-gtk3, libdbusmenu
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aether";
+  version = "2.0.0-dev15";
+  binaryName = "AetherP2P";
+  desktopName = "Aether";
```

Comment:
These two aren't used in the derivation. See `libPath` below. You can use a let binding instead.

Reply:
I addressed this, please resolve if you are fine with it.